.github/workflows/build-test-release.yml

```diff
@@ -129,6 +129,44 @@ jobs:
           path: |
             test-results-${{ matrix.splunk.version }}
 
+  test-splunk-external-wrong-hec-token:
```

Comment:
I'm sure there's a good reason behind moving this test here from `test-splunk-matrix` but I can't immediately see what this changes in how the test runs.

Reply:
PSA supports running tests using two types of Splunk instances (https://github.com/splunk/pytest-splunk-addon/blob/753adb5781c08d4110911cb135e5e1f443a53ec9/pytest_splunk_addon/splunk.py#L63 )
- external instance of Splunk (which we are using in CI) 
- docker instance of Splunk which will be spun by PSA 
While using docker option you can specify hec-token value (if not specified default value will be used for token creation) - and it will be later used while ingesting data. 
However, while using external option user can still pass value of hec token, and it might happen that it does not match the hec token existing in external Splunk instance (more details: https://splunk.atlassian.net/browse/ADDON-66599). 
So we need to check if hec token is valid for external option only. I removed this test from `test-splunk-matrix` since we use there docker instances of Splunk.

Reply:
Please consider including this as a part of existing `test-splunk-external` job in workflow